tests/Unit/Service/Validation/EntityValidatorTest.php

```diff
@@ -67,6 +67,9 @@ public function testValidate($expectedException, ValidationOptions $validationOp
             if ($expectedException !== null) {
                 $this->fail('Expected exception');
             }
+            if (method_exists($this, 'expectNotToPerformAssertions')) {
+                $this->expectNotToPerformAssertions();
+            }
```

Comment:
If we bump PHPUnit version to at least 7.5 we might not need this `if` clause, because this method was introduced in [PHPUnit 7.2](https://github.com/sebastianbergmann/phpunit/blob/7.2.0/ChangeLog-7.2.md).